### ISSUE TITLE
SOLVER: procedures

### ISSUE THREAD

**Issue by MichalKahun** [kahunmedical/kne#400]
adding patient finding - NORMAL procedure
(Example: Chest X-ray = NORMAL, colonoscopy = NORMAL)
Solver should decrease the suspicion of diseases in the DD according to the sensitivity of the procedure in the disease. 
Example of test case:
http://kahuneditor.s3-website-us-east-1.amazonaws.com/index.html#/w-kahun/caseEdit/CS.4048b458-9dc7-4132-842b-94b4ba7a68d3

**Comment by kahunmedicaltal** [comment]
@MichalKahun - adding NORMAL (as the procedure result) does make the disease less suspicious.  What makes you think this is not working

**Comment by shaharov** [comment]
Michal wants to set a single finding - colonscopy|normal without specifying a disease , then the solver should reduce the probability of all diseases that can be discovered with colonscopy

**Comment by kahunmedicaltal** [comment]
Can't we use panels for doing this?

**Comment by MichalKahun** [comment]
This is similar to panels. A panel needs to be created for each procedure, and each panel needs to include all the disorders that can be detected by the procedure.

**Comment by kahunmedicaltal** [comment]
Right. We can define that including a procedure in the panel is the same as including all diseases that can be identified by the procedure. I.E. if US is in the panel as we have a rules going to Appendicitis:US and Pancreatitis:US then Appendicitis:US and Pancreatitis:US would be automatically included in the panel

**Comment by MichalKahun** [comment]
#128
